src/groupByKey.ts

```diff
@@ -2,6 +2,14 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(items: [], key: string): GroupsMap<{}> {
+  const result: GroupsMap<{}> = {};
+
+  for (let i = 0; i < items.length; i += 1) {
```

Comment:
If this does not cause a linter error - replace `i += 1` with `i++` (If there is an error, write it under my comment)

Reply:
i++ linter error (Unary operator '++' used.)